WebRoot/css/zm.css

```diff
@@ -1667,7 +1667,31 @@ div.CompDragArea * {
 
 .ZmGrantRightsDialog .addrBubbleContainer,
 .ZmPreferencesPage .addrBubbleContainer {
-    width: 290px;
+	@GrantRightsDialogInputContainer@
+}
+
+.ZmGrantRightsDialog .WindowOuterContainer {
+	@GrantRightsDialogOuterContainer@
+}
+
+.ZmGrantRightsDialog #ZmGrantRightsDialog_name_row .Label {
+	@GrantRightsDialogEmailLabel@
+}
+
+.ZmGrantRightsDialog INPUT[type='text'] {
+	@GrantRightsDialogInput@
+}
+
+.ZmGrantRightsDialog .Label:empty {
+	display:none;
+}
+
+.ZmGrantRightsDialog .SendAs, .ZmGrantRightsDialog .SendObo {
+	padding-top:@SkinWrapperPadding@;
```

Comment:
don't think its a right use of skin **wrapper** padding value. 
Also better approach could be to apply line-height on the <tr> and not using padding on the <td>'s, this will keep the content vertically center as well.